scss/_header.scss

```diff
@@ -1,161 +1,161 @@
-    display: none;
-  }
-}
-
-// SLOGAN
-
-.header__slogan {
-  margin: 16% auto 3.3% auto;
-  padding: 0 2% 0 2%;
-  text-align: center;
-}
-
-.header__greet {
-  text-transform: capitalize;
-  font: italic 2rem $font-secondary;
-  margin-bottom: 2%;
-  @include respond-to(medium) {
-    font-size: 2.8rem;
-  }
-}
-
-.header__heading {
-  text-transform: uppercase;
-  font-size: 3rem;
-  font-weight: 700;
-  word-spacing: -5px;
-  @include respond-to(medium) {
-    font-size: 4.688rem;
-  }
-}
-
-// BTN
-
-.header__wrapper-btn {
-  display: flex;
-  margin: 0 auto;
-  max-width: 18.750rem;
-  width: 100%;
-  justify-content: center;
-}
-
-.header__btn {
-  @include link-style;
-  @include button-big;
-  // fallback for old browsers that have problems with % padding on flex items
-  padding: 1.4rem 3rem;
-  padding: 7.5% 16%;
+.header {
+  padding: 3.2% 0 11.5% 0;
+  background:
+  linear-gradient(
+    rgba(0, 0, 0, 0.5),
+    rgba(0, 0, 0, 0.5)
+  ),
+  url(./../images/header/bgimage-small.jpg) center/cover no-repeat;
+  color: #fff;
+  font-family: $font-main;
+  @include respond-to(medium) {
+    background:
+    linear-gradient(
+      rgba(0, 0, 0, 0.5),
+      rgba(0, 0, 0, 0.5)
+    ),
+    url(./../images/header/bgimage.jpg) center/cover no-repeat;
+  }
+}
+
+.header__width {
+  @include section-width;
+}
+
+// NAV
+
+.header__nav {
+  display: flex;
+  align-items: center;
+  position: relative;
+  justify-content: space-between;
+  padding: 0 10.3% 0 11.4%;
+}
+
+.header__list {
+  display: none;
+  list-style-type: none;
+  right: 0;
+  left: 0;
+  @include respond-to(medium) {
+    display: flex;
+    justify-content: space-between;
+    max-width: 480px;
+    width: 100%;
+  }
+}
+
+.header__item {
+  position: relative;
+  margin: 0;
+  padding: 5%;
+  @include respond-to(medium) {
+    padding: 0;
+    margin: 0 0 0 -1.5%;
+  }
+}
+
+.header__item:before {
+  @include respond-to(medium) {
+    content: "";
+    position: absolute;
+    width: 100%;
+    height: 2px;
+    bottom: 0;
+    left: 0;
+    background-color: #fff;
+    visibility: hidden;
+    transform: scaleX(0);
+    transition: all 0.3s ease-in-out 0s;
+  }
+}
+
+.header__item:hover:before {
+  @include respond-to(medium) {
+    visibility: visible;
+    transform: scaleX(1);
+  }
+}
+
+.header__link {
+  @include link-style;
+  text-transform: uppercase;
+  font-size: 1rem;
+  padding: 4%;
+}
+
+#menu-toggle {
+  display: none;
+  &:checked ~ .header__list {
+    // mixin respond-to uses min-width
+    @media screen and (max-width: 768px) {
+      display: block;
+      width: 100%;
+      text-align: center;
+      position: absolute;
+      top: 100%;
+      background-color: rgba(47, 47, 50, 0.95);
+    }
+  }
+  &:not(:checked) ~ .header__list {
+    // mixin respond-to uses min-width
+    @media screen and (max-width: 768px) {
+      display: none;
+    }
+  }
+}
+
+.header__toggle {
+  font-size: 1.5rem;
+  &:hover {
+    cursor: pointer;

```

Comment:
If it's just `cursor` property you can move it up to selector, not using hover state.